eos/maths/legendre-polynomial-vector.hh

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright (c) 2024 Florian Herren
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#ifndef EOS_GUARD_EOS_MATHS_LEGENDRE_POLYNOMIAL_VECTOR_HH
+#define EOS_GUARD_EOS_MATHS_LEGENDRE_POLYNOMIAL_VECTOR_HH 1
+
+#include <array>
+#include <vector>
+#include <complex>
+
+#include <boost/math/special_functions/legendre.hpp>
```

Comment:
Added a new commit where I add it in doc/installation.rst. Does it need to go elsewhere?

Reply:
Since the test cases pass in our docker images: probably not. Nevertheless, it would be nice to include it expliciltly when building Boost. I have opened an issue: eos/docker.io/issues/19